dist/index.js

```diff
@@ -1073,7 +1073,7 @@ function createOrUpdateBranch(git, commitMessage, base, branch, branchRemoteName
         // Save the working base changes to a temporary branch
         const tempBranch = uuid_1.v4();
         yield git.checkout(tempBranch, 'HEAD');
-        // Commit any uncomitted changes
+        // Commit any uncommitted changes
```

Comment:
I can't figure out where this code comes from.

Reply:
This is my code. It's file `src/create-or-update-branch.ts`. This PR already corrects this occurrence.
The `dist/index.js` file is compiled from my code plus all of the dependencies it's using.